webapp/src/main/java/org/exoplatform/officeonline/portlet/EditorPortlet.java

```diff
@@ -0,0 +1,94 @@
+package org.exoplatform.officeonline.portlet;
+
+import java.io.IOException;
+
+import javax.jcr.RepositoryException;
+import javax.portlet.GenericPortlet;
+import javax.portlet.PortletException;
+import javax.portlet.PortletRequestDispatcher;
+import javax.portlet.RenderMode;
+import javax.portlet.RenderRequest;
+import javax.portlet.RenderResponse;
+
+import org.exoplatform.container.ExoContainer;
+import org.exoplatform.container.ExoContainerContext;
+import org.exoplatform.officeonline.AccessToken;
+import org.exoplatform.officeonline.EditorConfig;
+import org.exoplatform.officeonline.EditorService;
+import org.exoplatform.officeonline.RequestInfo;
+import org.exoplatform.officeonline.WOPIService;
+import org.exoplatform.officeonline.exception.OfficeOnlineException;
+import org.exoplatform.services.log.ExoLogger;
+import org.exoplatform.services.log.Log;
+import org.exoplatform.web.application.JavascriptManager;
+import org.exoplatform.web.application.RequireJS;
+import org.exoplatform.webui.application.WebuiRequestContext;
+import org.exoplatform.ws.frameworks.json.impl.JsonException;
+
+/**
+ * The Class EditorPortlet.
+ */
+public class EditorPortlet extends GenericPortlet {
+
+  /** The Constant DEFAULT_ACTION. */
+  private static final String DEFAULT_ACTION = "edit";
+
+  /** The Constant LOG. */
+  private static final Log    LOG            = ExoLogger.getLogger(EditorPortlet.class);
+
+  /** The onlyoffice. */
+  private EditorService       editorService;
+
+  /** The onlyoffice. */
+  private WOPIService         wopiService;
+
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public void init() throws PortletException {
+    super.init();
+    ExoContainer container = ExoContainerContext.getCurrentContainer();
```

Comment:
As it's in the portlet code, it is more correct to use PortalContainer.getInstance(). This will tell that only a portal container expected here (not a root one).

Reply:
Fixed in https://github.com/exo-addons/office-online/pull/4/commits/02ae41be8f07d7ad2cc22669394cc34855d37e52

Reply:
I don't see use of `PortalContainer.getInstance()` in the commit.

Reply:
Oops, I accidentally changed it in Onlyoffice project (was also opened in Eclipse). 
Used in next commit: https://github.com/exo-addons/office-online/pull/4/commits/3da599587ed379291fd3d9f62c27447ecd8fb021